site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,438 @@
+
+> You can adapt this codelab to use any of the boards listed under hardware in the prerequisite section.
+
+![EFR32MG Hardware setup](./img/Silabs/EFR32MG_hardware_setup.jpg)
+
+### What you'll learn
+
+* How to create an OpenThread project using Silicon Labs Simplicity Studio IDE.
+* How to build and flash OpenThread CLI binaries to Silicon Labs radio boards.
+* How to setup a Raspberry Pi 3B+ or greater as an OpenThread Border Router (OTBR) using Docker.
+* How to create a Thread network on the OTBR.
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+
+
+## Prerequisites
+
+Hardware:
+
+* 3 EFR32MGxx radio boards - you can use any combination of these devices. I used a BRD4166A as the RCP and two BRD4168As as Full Thread Devices.
+  
+  * EFR32MG12 (BRD**4161A**, BRD**4166A**, BRD**4170A**, BRD**4304**A)
+  * EFR32MG13 (BRD**4168A**)
+  * EFR32MG21 (BRD**4180A**, BRD**4180B**)
+  * If you are starting fresh, you can get one of the [EFR32 Thread starter kit](https://www.silabs.com/wireless/thread) which has the boards listed above.
+  
+* BRD4001A: Wireless starter main boards (WSTK) to host the radio boards. Except for BRD4166A, all the radio boards need a starter main board each. Mini USB cables to connect and power the main boards or micro USB cable for BRD4166A.
+
+  > Ensure the 3 way switch on your main board is in AEM position as shown below. The BRD4166A doesn't have this switch.
+
+  ![WSTK AEM](./img/Silabs/WSTK_AEM.jpg)
+
+* A Raspberry Pi 3B+ or greater with [Raspbian Stretch Lite OS image](http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip) or [Raspbian Stretch with Desktop](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip) which is connected to the internet over Ethernet. We configure this as an OT Border Router.
+
+* A Windows/Linux/Mac Host system with at least 2 USB ports and an internet connection. Check the hardware and OS requirements at [SSv5](https://www.silabs.com/developers/simplicity-studio).
+
+* At least one Ethernet cable for connecting the Raspberry Pi to Internet. The WSTKs also support debug and flashing over IP so, optionally, additional Ethernet cables can be used to connect the WSTK to your host system via a Ethernet Switch.
+
+Software:
+
+* [Simplicity Studio v5](https://www.silabs.com/developers/simplicity-studio) installed and updated on the Windows/Linux/Mac Host system with
+
+  * GNU ARM v7.2.1 toolchain 
+
+  * Gecko SDK Suite 3.1.0 or later and Silicon Labs OpenThread SDK.
+
+    > The IAR Embedded Workbench for ARM (IAR EWARM) Compiler is currently not supported with OpenThread.
+
+
+
+## Getting started
```

Comment:
> This step is quite lengthy, I suggest we break it up into the following so it's more digestible:
> 
> Hardware setup
> Firmware setup - demos
> Firmware setup - projects
> 
> Ideally in a codelab the user shouldn't jump around or skip steps, though, so it would be better to just have one setup method. Would you be willing to just have one method of firmware setup and remove the other, leaving it as an exercise for the reader?

@Vyrastas  I have reorganised the secion into Hardware and Firmware setup.  Firmware setup - demos is put as an optional exercise in the end. Is it possible to link this optional section in the earlier step under Firmware set up point 2 Demos?